gno.land/pkg/integration/node_testing.go

```diff
@@ -178,8 +178,9 @@ func DefaultTestingTMConfig(gnoroot string) *tmcfg.Config {
 
 	tmconfig := tmcfg.TestConfig().SetRootDir(gnoroot)
 	tmconfig.Consensus.WALDisabled = true
+	tmconfig.Consensus.SkipTimeoutCommit = true
 	tmconfig.Consensus.CreateEmptyBlocks = true
-	tmconfig.Consensus.CreateEmptyBlocksInterval = time.Duration(0)
+	tmconfig.Consensus.CreateEmptyBlocksInterval = time.Millisecond * 100
```

Comment:
I think this is a major part of the speed-up -- now test nodes don't have 5s timeouts for each block

Reply:
it definitely is, but parallel processing also have a huge impact, if you run everything in memory (and so, sequentailly by default): `INMEMORY_TS=true go test -v -count=1 ./pkg/integration/...` it will double test time.

Also i forgot to mention it but `gnoclient` was also taking a HUGE amount of time by loading everything in example folder on each tests.